cnn_framework/uncertainty_test.py

```diff
@@ -0,0 +1,29 @@
+import keras.backend as K
+import numpy as np
+from keras.layers import Input
+from uncertainty import RandomMask, EnsembleModel
+import time
+import unittest
+
+class TestRandomMask(unittest.TestCase):
+    def testRandomMask(self):
```

Comment:
Please change the names of these methods to use underscores and lowercase, e.g., `test_random_mask`.

Reply:
Done